frontend/catalyst/compiler.py

```diff
@@ -254,6 +261,32 @@ def get_default_flags():
         else:
             pass  # pragma: nocover
 
+        ### rpath and -L: custom calls
+        lib_path_flags += [
+            f"-Wl,-rpath,{DEFAULT_CUSTOM_CALLS_LIB_PATH}",
+            f"-L{DEFAULT_CUSTOM_CALLS_LIB_PATH}",
+        ]
+        file_prefix = "libcustom_calls"
+        file_extension = ".so"
+        search_pattern = path.join(DEFAULT_CUSTOM_CALLS_LIB_PATH, f"{file_prefix}*{file_extension}")
+        custom_calls_so_file = path.basename(glob.glob(f"{search_pattern}")[0])
+        custom_calls_so_flag = f"-l{custom_calls_so_file[3:-len(file_extension)]}"
```

Comment:
I could not find a simple way to add the name of the created shared library in the setup.py file. I think we could set the variable `EXT_SUFFIX` to nothing, but I am not sure it would not have any other impact.

Reply:
Looks the question was also asked here: https://stackoverflow.com/a/60285245

Seems we can just override the `get_ext_filename` method in the `build_ext` subclass.

Reply:
It works, thanks!